### DISCUSSION TITLE
Use the code for my own org?

### DISCUSSION THREAD

**Discussion by CholoTook** [opensafely/documentation#613] [Installing OpenSAFELY] [answered]
Sorry if I'm being stupid, I can't find how to download and run the code for my own collaborations on our own data?

**Comment by bloodearnest** [comment, marked as answer]
Hi

OpenSAFELY is a platform with a collection of tools that work together to provide researchers access to some sensitive datasets. It is not a generic standalone data processing tool - it supports a few very specific patient record datasets (although we are adding more over time).

Some of these tools are designed to be downloaded and run locally using with dummy data to test your analytic code (e.g. opensafely-cli and the various docker images), some are services that we run centrally as part of the the platform (e.g. job-runner, job-server) that enable approved researchers to run code against sensitive datasets in a secure and auditable manner.

If you want to be able to run against your own datasets, you would need to be some significant development work to add support for it. We are currently working on making this a much simpler prospect, but it would still involve integration work.

Does that help clarify things?

**Comment by CholoTook** [comment]
Thank you, yes, that's clear now.

I wasn't sure from the homepage if I could load my own data (e.g. from a consortium collaboration) and use this as a platform to allow partner access to the data, for example.

There seem to be commercial tools out there to do this, e.g. DNANexus, but I'm not sure if the cost is right for us.

It could be interesting to look at how much of a market for 'private' data there is out there, especially if it could be rolled into the central datasets easily as consortia data becomes public.


Many thanks,
Dan.
